application/config/init.php

```diff
@@ -28,7 +28,7 @@
 	'base_url'    => '/',
 	'index_file'  => FALSE,
 	'charset'     => 'utf-8',
-	'errors'      => TRUE,
+	'errors'      => false,
```

Comment:
what does this do? (or, why was that True before)

Reply:
It shouldn't have changed! Good catch. So it's stupid, Kohana error logging is broken because it's basically decayed. If this is set to True Kohana intercept the error message but fails to actually except it properly. By disabling it, the errors pass straight through. I forgot to put it back to True for merge!

Reply:
ahh good. Thanks for explaining 👍